app/assets/javascripts/discourse/app/lib/plugin-api.gjs

```diff
@@ -966,7 +968,7 @@ class PluginApi {
    **/
   addHeaderPanel(name, toggle, transformAttrs) {
     deprecated(
-      "addHeaderPanel has been removed. Use api.addToHeaderIcons instead.",
+      "addHeaderPanel has been removed. Use api.headerIcons instead.",
```

Comment:
I think we should say 'will be removed' instead of 'has been removed' 
```suggestion
      "addHeaderPanel will be removed as part of the glimmer header upgrade. Use api.headerIcons instead.",
```